dist/routable-component.js

```diff
@@ -184,7 +184,7 @@ var routableComponent =
 	        for (var _iterator2 = (0, _getIterator3.default)(this.routes), _step2; !(_iteratorNormalCompletion2 = (_step2 = _iterator2.next()).done); _iteratorNormalCompletion2 = true) {
 	          var route = _step2.value;
 	
-	          if (route.from === name) this._attach_route_event(route);
+	          if (route.from === name) this._attachRouteEvent(route);
```

Comment:
__[eslint]__
* Error - Mixed spaces and tabs. (no-mixed-spaces-and-tabs)
* Error - Expected indentation of 2 space characters but found 0. (indent)

<!-- Comment by [SideCI](https://sideci.com) -->